code/controllers/subsystem/spatial_gridmap.dm

```diff
@@ -349,7 +349,9 @@ SUBSYSTEM_DEF(spatial_grid)
 ///find the spatial map cell that target belongs to, then add target's important_recusive_contents to it.
 ///make sure to provide the turf new_target is "in"
 /datum/controller/subsystem/spatial_grid/proc/enter_cell(atom/movable/new_target, turf/target_turf)
-	if(!target_turf || !new_target?.important_recursive_contents)
+	if(!initialized)
+		return
+	if(!target_turf || QDELING(new_target) || !new_target?.important_recursive_contents)
```

Comment:
Don't we want QDELETED here?

Reply:
technically the null check is handled by !new_target?.important_recursive_contents but i made it QDELETED after i separated it into its own check